src/resources/views/show.blade.php

```diff
@@ -15,19 +15,50 @@
 
 @section('content')
 	@if ($crud->hasAccess('list'))
-		<a href="{{ url($crud->route) }}"><i class="fa fa-angle-double-left"></i> {{ trans('backpack::crud.back_to_all') }} <span>{{ $crud->entity_name_plural }}</span></a><br><br>
+		<a href="{{ url($crud->route) }}"><i class="fa fa-angle-double-left"></i> {{ trans('backpack::crud.back_to_all') }} <span class="text-lowercase">{{ $crud->entity_name_plural }}</span></a><br><br>
 	@endif
 
 	<!-- Default box -->
 	  <div class="box">
 	    <div class="box-header with-border">
 	      <h3 class="box-title">
             {{ trans('backpack::crud.preview') }}
-            <span>{{ $crud->entity_name }}</span>
+            <span class="text-lowercase">{{ $crud->entity_name }}</span>
           </h3>
 	    </div>
 	    <div class="box-body">
-	      {{ dump($entry) }}
+			<table class="table table-striped table-bordered">
+		        <tbody>
+		        @foreach ($crud->columns as $column)
+		            <tr>
+		                <td>
+		                    <strong>{{ $column['label'] }}</strong>
+		                </td>
+							@if (!isset($column['type']))
+		                      @include('crud::columns.text')
+		                    @else
+		                      @if(view()->exists('vendor.backpack.crud.columns.'.$column['type']))
+		                        @include('vendor.backpack.crud.columns.'.$column['type'])
+		                      @else
+		                        @if(view()->exists('crud::columns.'.$column['type']))
+		                          @include('crud::columns.'.$column['type'])
+		                        @else
+		                          @include('crud::columns.text')
+		                        @endif
+		                      @endif
+		                    @endif
+		            </tr>
+		        @endforeach
+				@if ($crud->buttons->where('stack', 'line')->count())
+					<tr>
+						<td><strong>Actions</td>
```

Comment:
You sure that doesn't have a translation key?

Reply:
Fixed! Thank you.